XenAdmin/Wizards/PatchingWizard/PlanActions/ApplyXenServerPatchPlanAction.cs

```diff
@@ -67,7 +68,8 @@ protected override void RunWithSession(ref Session session)
             }
             else
             {
-                log.ErrorFormat("Mapping not found for patch {0} on master {1}", xenServerPatch.Uuid, master.uuid);
+                if (xenServerPatch != null && master != null)
+                    log.ErrorFormat("Mapping not found for patch {0} on master {1}", xenServerPatch.Uuid, master.uuid);

```

Comment:
What if XenServerPatch and/or master is null here... would you still want to log something?